notebooks/stage/gz_to_raw.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>I don't like this. Spawning a new shell process and executing a command for each file doesn't seem like a best idea to me. Keep in mind we have no control if the <code>gzip</code>  executable is even available on the system. This may later cause troubles in automation - when it's packaged into a container which doesn't feature this executable. Why don't we use python gzip stdlib package? It's this simple:</p><p> </p><pre class="ql-syntax" spellcheck="false">import gzip
import shutil
with gzip.open('file.mbox.gz', 'rb') as f_in:
    with open('file.mbox', 'wb') as f_out:
        shutil.copyfileobj(f_in, f_out)
</pre><p> </p><p>No need to spawn new processes, no need to install additional system packages or python packages...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing_list_analysis_toolkit/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Thanks @tumido , your right this is much better. </p>